src/NServiceBus.Core/obsoletes.cs

```diff
@@ -881,7 +881,7 @@ public interface IWantToRunWhenConfigurationIsComplete
 namespace NServiceBus.Faults
 {
     [ObsoleteEx(
-        Message = "IManageMessageFailures is no longer an extension point. If full control over what happens when a message fails (including retries) is required then override the MoveFaultsToErrorQueue behavior. To get notified when messages are being moved use BusNotifications.Errors.MessageSentToErrorQueue.Subscribe(e=>{}) ",
+        Message = "IManageMessageFailures is no longer an extension point. To achieve a similar behavior please review the version 5 to 6 upgrade guide",
```

Comment:
Could we have something like?

> IManageMessageFailures is no longer an extension point. To achieve full control over what happens when a message fails review the version 5 to 6 upgrade guide. 

I think it would be more precise than just saying

> achieve a similar behavior

and it also removes the word "please" ;)